masakari-controller/controller/masakari_util.py

```diff
@@ -569,89 +606,91 @@ def _fetch_project_id(self):
 
         return projects[0].id
 
+    @log_process_begin_and_end.output_log
```

Comment:
On second thought, I think that this decorator isn't necessary.
You are right.
I remove this decorator.
And, I check as to whether there is other unnecessary decorators.